test/10_before_test/cibuildwheel_test.py

```diff
@@ -0,0 +1,22 @@
+import os
+import utils
+
+def test():
+    project_dir = os.path.dirname(__file__)
+
+    # build the wheels
+    actual_wheels = utils.cibuildwheel_run(project_dir, add_env={
+        # write python version information to a temporary file, this is
+        # checked in setup.py
+        'CIBW_BEFORE_TEST': '''python -c "import sys; open('/tmp/pythonversion.txt', 'w').write(sys.version)" && python -c "import sys; open('/tmp/pythonexecutable.txt', 'w').write(sys.executable)"''',
```

Comment:
The `pythonversion.txt` stuff is overkill for this test, I think. Good to check we're in the right virtualenv, though. Maybe we could do it just with `pythonexecutable.txt`?

Reply:
Why it is overkill here and not overkill in `CIBW_BEFORE_BUILD` test?

Reply:
I think the `pythonversion.txt` check in BEFORE_BUILD is there through an abundance of caution (paranoia?) about which version of Python is running. Maybe it's not necessary there either. But we certainly don't need it here.